Add brief install instructions for the NESTML package in our ppa (#3037)
nest/nest-simulator: doc/htmldoc/installation/user.rst

     sudo apt-get install nest

>    Or install NEST with `NESTML <https://nestml.readthedocs.io/en/latest/index.html>`_

Reviewer: ```suggestion
Or install NEST with `NESTML <https://nestml.readthedocs.io/en/latest/index.html>`_
```

     sudo apt-get install nest

> 3. Optional: Install `NESTML <https://nestml.readthedocs.io/en/latest/index.html>`_

Reviewer: ```suggestion
   Or install NEST with `NESTML <https://nestml.readthedocs.io/en/latest/index.html>`_
```

.. code-block:: bash

    sudo apt install nest python3-nestml
    python3 -m pip install --upgrade odetoolbox pygsl antlr4-python3-runtime==4.10

> 4. Set the environment

Reviewer: ```suggestion
3. Set the environment
```
